test/e2e_tests/utils/userActions.ts

```diff
@@ -112,3 +113,95 @@ export const handleAppLockState = async (pageManager: PageManager, appLockPassCo
     }
   }
 };
+
+/**
+ * Logs in a user and handles initial setup
+ */
+export async function loginAndSetup(user: User, pageManager: PageManager) {
```

Comment:
Please prefer the fixtures `createUser()` and `createPage()` with the plugins `withLogin` instead of creating a new util. You can find examples on how to use them in `edit.spec.ts`